cmd/queue/main.go

```diff
@@ -316,15 +312,16 @@ func buildServer(ctx context.Context, env config, healthState *health.State, pro
 		composedHandler = tracing.HTTPSpanMiddleware(composedHandler)
 	}
 
-	composedHandler = health.ProbeHandler(healthState, probeContainer, tracingEnabled, composedHandler)
-	composedHandler = network.NewProbeHandler(composedHandler)
-	// We might sometimes want to capture the probes/healthchecks in the request
-	// logs. Hence we need to have RequestLogHandler to be the first one.
+	var healthcheckHandler http.Handler
+	healthcheckHandler = health.ProbeHandler(probeContainer, tracingEnabled, healthcheckHandler)
 	if env.ServingEnableRequestLog {
-		composedHandler = requestLogHandler(logger, composedHandler, env)
+		healthcheckHandler = requestLogHandler(logger, healthcheckHandler, env)
```

Comment:
composedHandler needs to be wrapped with the requestLogHandler here

Reply:
its required in both? the `RequestLogTest` is passing without it

Reply:
Yeah we should - otherwise normal requests to the user container aren't logged